src/RecoveryServices/RecoveryServices/help/Set-AzRecoveryServicesAsrReplicationProtectedItem.md

```diff
@@ -181,6 +184,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DiskTag
+Specify the tags for the disks of the VM.
+
```

Comment:
please update the description so that user knows for what scenarios, this is applicable. Same for all tags